test/unit/option_spec.lua

```diff
@@ -0,0 +1,64 @@
+
+local helpers = require("test.unit.helpers")
+
+local to_cstr = helpers.to_cstr
+local eq      = helpers.eq
+
+local option = helpers.cimport("./src/nvim/option.h")
+local globals = helpers.cimport("./src/nvim/globals.h")
+
+describe('option functions', function()
+  local check_ff_value = function(ff)
+    return option.check_ff_value(to_cstr(ff))
+  end
+
+  describe('check_ff_value string', function()
+
+    it('should view empty string as valid', function()
+      eq(1, check_ff_value(""))
+    end)
+
+    it('should view "unix" as valid', function()
+      eq(1, check_ff_value("unix"))
+    end)
+    
+    it('should view "dos" as valid', function()
+      eq(1, check_ff_value("dos"))
+    end)
+
+    it('should view "mac" as valid', function()
+      eq(1, check_ff_value("mac"))
+    end)
+
+    it('should view "foo" as invalid', function()
+      eq(0, check_ff_value("foo"))
+    end)
+  end)
+
+  local get_sts_value = function()
+    return option.get_sts_value()
+  end
+
+  describe('get_sts_value', function()
+    it('should return "softtabstop" when it is non-negative"', function()
+      globals.curbuf.b_p_sts = 5
+      eq(5, get_sts_value())
+      globals.curbuf.b_p_sts = 0
+      eq(0, get_sts_value())
+    end)
+
+    it('should use "effective shiftwidth" when "softtabstop" is negative"', function()
```

Comment:
throughout the Vim source and docs, options are always surrounded with single-quotes, e.g. `'softtabstop'`. So that can be done here:

    it([[uses "effective shiftwidth" when 'softtabstop' is negative]], function()